diff --git a/modules/completion/vertico/config.el b/modules/completion/vertico/config.el
index ddad6109..e56b18e8 100644
--- a/modules/completion/vertico/config.el
+++ b/modules/completion/vertico/config.el
@@ -39,11 +39,11 @@ overrides `completion-styles' during company completion sessions.")
         :map vertico-map
         "M-RET" #'vertico-exit-input
         "C-SPC" #'+vertico/embark-preview
-        "C-j"   #'vertico-next
-        "C-M-j" #'vertico-next-group
-        "C-k"   #'vertico-previous
-        "C-M-k" #'vertico-previous-group
-        "C-h" (cmds! (eq 'file (vertico--metadata-get 'category)) #'vertico-directory-up)
+        "C-k"   #'vertico-next
+        "C-M-k" #'vertico-next-group
+        "C-h"   #'vertico-previous
+        "C-M-h" #'vertico-previous-group
+        "C-j" (cmds! (eq 'file (vertico--metadata-get 'category)) #'vertico-directory-up)
         "C-l" (cmds! (eq 'file (vertico--metadata-get 'category)) #'+vertico/enter-or-preview))
 
   ;; Cleans up path when moving directories with shadowed paths syntax, e.g.
diff --git a/modules/config/default/+evil-bindings.el b/modules/config/default/+evil-bindings.el
index 5cd3be65..37d0439a 100644
--- a/modules/config/default/+evil-bindings.el
+++ b/modules/config/default/+evil-bindings.el
@@ -8,8 +8,8 @@
         "C-a" #'evil-beginning-of-line
         "C-b" #'evil-backward-char
         "C-f" #'evil-forward-char
-        :gi "C-j" #'next-complete-history-element
-        :gi "C-k" #'previous-complete-history-element)
+        :gi "C-k" #'next-complete-history-element
+        :gi "C-h" #'previous-complete-history-element)
 
   (define-key! :keymaps +default-minibuffer-maps
     [escape] #'abort-recursive-edit
@@ -21,17 +21,17 @@
     "C-z"    (cmd! (ignore-errors (call-interactively #'undo))))
 
   (define-key! :keymaps +default-minibuffer-maps
-    "C-j"    #'next-line
-    "C-k"    #'previous-line
-    "C-S-j"  #'scroll-up-command
+    "C-k"    #'next-line
+    "C-h"    #'previous-line
+    "C-S-h"  #'scroll-up-command
     "C-S-k"  #'scroll-down-command)
   ;; For folks with `evil-collection-setup-minibuffer' enabled
   (define-key! :states 'insert :keymaps +default-minibuffer-maps
-    "C-j"    #'next-line
-    "C-k"    #'previous-line)
+    "C-k"    #'next-line
+    "C-h"    #'previous-line)
   (define-key! read-expression-map
-    "C-j" #'next-line-or-history-element
-    "C-k" #'previous-line-or-history-element))
+    "C-k" #'next-line-or-history-element
+    "C-h" #'previous-line-or-history-element))
 
 
 ;;
@@ -99,8 +99,8 @@
        :nv "S" #'evil-mc-skip-and-goto-prev-match
        :nv "c" #'evil-mc-skip-and-goto-next-cursor
        :nv "C" #'evil-mc-skip-and-goto-prev-cursor
-       :nv "j" #'evil-mc-make-cursor-move-next-line
-       :nv "k" #'evil-mc-make-cursor-move-prev-line
+       :nv "k" #'evil-mc-make-cursor-move-next-line
+       :nv "h" #'evil-mc-make-cursor-move-prev-line
        :nv "m" #'evil-mc-make-all-cursors
        :nv "n" #'evil-mc-make-and-goto-next-cursor
        :nv "N" #'evil-mc-make-and-goto-last-cursor
@@ -136,9 +136,9 @@
          "C-w"     nil  ; don't interfere with `evil-delete-backward-word'
          "C-n"     #'company-select-next
          "C-p"     #'company-select-previous
-         "C-j"     #'company-select-next
-         "C-k"     #'company-select-previous
-         "C-h"     #'company-show-doc-buffer
+         "C-k"     #'company-select-next
+         "C-h"     #'company-select-previous
+         "C-j"     #'company-show-doc-buffer
          "C-u"     #'company-previous-page
          "C-d"     #'company-next-page
          "C-s"     #'company-filter-candidates
@@ -151,8 +151,8 @@
         (:map company-search-map  ; applies to `company-filter-map' too
          "C-n"     #'company-select-next-or-abort
          "C-p"     #'company-select-previous-or-abort
-         "C-j"     #'company-select-next-or-abort
-         "C-k"     #'company-select-previous-or-abort
+         "C-k"     #'company-select-next-or-abort
+         "C-h"     #'company-select-previous-or-abort
          "C-s"     #'company-filter-candidates
          [escape]  #'company-search-abort)))
 
@@ -178,10 +178,10 @@
         "C-S-n"    #'helm-next-source
         "C-S-p"    #'helm-previous-source
         (:when (modulep! :editor evil +everywhere)
-         "C-j"    #'helm-next-line
-         "C-k"    #'helm-previous-line
-         "C-S-j"  #'helm-next-source
-         "C-S-k"  #'helm-previous-source)
+         "C-k"    #'helm-next-line
+         "C-h"    #'helm-previous-line
+         "C-S-k"  #'helm-next-source
+         "C-S-h"  #'helm-previous-source)
         "C-u"      #'helm-delete-minibuffer-contents
         "C-s"      #'helm-minibuffer-history
         ;; Swap TAB and C-z
@@ -197,7 +197,7 @@
         [C-return] #'helm-ff-run-switch-other-window
         "C-w"      #'helm-find-files-up-one-level
         (:when (modulep! :editor evil +everywhere)
-          "C-h"    #'helm-find-files-up-one-level
+          "C-j"    #'helm-find-files-up-one-level
           "C-l"    #'helm-execute-persistent-action))
        (:after helm-locate :map helm-generic-files-map
         [C-return] #'helm-ff-run-switch-other-window)
diff --git a/modules/lang/org/config.el b/modules/lang/org/config.el
index 9869fd5c..78f02f9c 100644
--- a/modules/lang/org/config.el
+++ b/modules/lang/org/config.el
@@ -1069,9 +1069,9 @@ between the two."
          "b" #'org-tree-to-indirect-buffer
          "c" #'org-clone-subtree-with-time-shift
          "d" #'org-cut-subtree
-         "h" #'org-promote-subtree
-         "j" #'org-move-subtree-down
-         "k" #'org-move-subtree-up
+         "j" #'org-promote-subtree
+         "k" #'org-move-subtree-down
+         "h" #'org-move-subtree-up
          "l" #'org-demote-subtree
          "n" #'org-narrow-to-subtree
          "r" #'org-refile

